gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobufdataset.cpp

```diff
@@ -307,11 +311,17 @@ bool OGRFlatGeobufDataset::OpenFile(const char* pszFilename, VSILFILE* fp, bool
 
     CPLDebug("FlatGeobuf", "Features start at offset (%lu)", static_cast<long unsigned int>(offset));
 
-    auto poLayer = std::unique_ptr<OGRFlatGeobufLayer>(
-        new OGRFlatGeobufLayer(header, buf.release(), pszFilename, fp, offset));
+    CPLDebug("FlatGeobuf", "Creating OGRFlatGeobufLayer");
```

Comment:
you'd probably want to use CPLDebugOnly() in a number of places to have such messages only for debug builds as they are likely not very interesting for production builds